NMDSdevopsServiceAdm/DataEngineering
# Add imputation into estimate_missing script

# Description
Imputation now outputs one column which is a combination of known data, extrapolated and interpolated data - the column name created contains the columns on the data used to create it.

Function called twice
- once for NR using filled posts and the rolling average model
- once for CH using ratio and the ratio rolling average model

# How to test
Data check running in branch - compare to previous outputs.
Check what happens to locations who switch between CH and NR.

[Trello ticket](https://trello.com/c/o0jObcV4/802-replace-current-extrap-interp-with-new-combined-version-in-the-pipeline)

# Developer checklist
